\documentclass{bioinfo}
\copyrightyear{2015} \pubyear{2015}

\access{Advance Access Publication Date: Day Month Year}
\appnotes{Manuscript Category}

\begin{document}
\firstpage{1}

\subtitle{Original Paper}

\title[An analysis of the overlap between DNA methylation and gene expression patterns during maternal immune activation]{An analysis of the overlap between DNA methylation and gene expression patterns during maternal immune activation}
\author[Breen \textit{et~al}.]{Breen, C\,$^{\text{\sfb}*}$}
\address{$^{\text{\sf}}$School of Mathematics, Statistics and Applied Mathematics, National University of Ireland, Galway, H91 TK33, Ireland}

\corresp{$^\ast$To whom correspondence should be addressed.}

\abstract{\textbf{Motivation:} Schizophrenia is one of many neurodevelopmental disorders associated with foetal exposure to a maternal inflammatory immune response. Maternal pro-inflammatory signals during neurodevelopment may cause permanent epigenetic changes that mediate the long-lasting effect of prenatal insult. This paper aims to explore the connection between DNA methylation and gene expression after maternal immune activation at the level of individual overlapping genes, commonly enriched biological pathways, and commonly enriched neuronal cell types.\\
\textbf{Results:} There is promising consensus between the two datasets, including pathways and cell types involved in neurogenesis and neurodevelopment. However, a statistically significant connection could not largely be established. The impact of DNA methylation on neurodevelopment is an encouraging direction for future functional study into the role of  on neurodevelopment. \\
\textbf{Availability:} The developed method is available on github: github.com/ConorBreen959/MSc\_Project.\\
\textbf{Contact:} \href{c.breen12@nuigalway.ie}{c.breen12@nuigalway.ie}\\
\textbf{Supplementary information:} Supplementary data are available in the accompanying supplementary files.}

\maketitle

\begin{introduction}
\section{Introduction}

\begin{figure*}[!ht]
\centering
\includegraphics[width = \linewidth]{Cell_Type_Taxonomy}
\caption{Zeisel et al, 2018. This graphic was taken from the source paper for the single cell expression dataset used for the cell type enrichment analysis. Part A of the graphic shows each brain region that was sampled in the study. Part B is the output of gt-SNE dimensionality reduction and clustering used by the authors to identify and visualise the single cell types present. Part C is then a constructed dendrogram giving detailed information on each of the cell types and their relationships to each other. This dendrogram was later used as a template guide for plotting and interpreting the EWCE results.}\label{fig:01}
\end{figure*}

Schizophrenia is a severe neuropsychiatric disorder with profound impact on patients' abilities to function in society \citep{joyce_cognitive_2007, american_psychiatric_association_diagnostic_2013}. This disorder, existing on a spectrum of highly variable symptoms \citep{guloksuz_slow_2018}, is likely caused by an interplay between genetic and environmental risk factors. There is a wealth of research detailing the associated genetic risk variants \citep{ripke_biological_2014, marshall_contribution_2017, walsh_rare_2008}, however given the missing heritability identified in family and twin studies, exposure to environmental factors likely plays a key role. Schizophrenia, and related conditions such as autism and bipolar disroder, are often referred to as neurodevelopmental disorders, in that they arise due to prenatal and early postnatal exposures which alter the normal trajectory of neurodevelopment \citep{fatemi_neurodevelopmental_2009}. One such event is foetal exposure to maternal immune activation. Prenatal infection is well established as increasing the risk of schizophrenia and related disorders, with a wide variety of pathogens \citep{sorensen_association_2009, babulas_prenatal_2006, mortensen_toxoplasma_2007, buka_maternal_2008}. Given the disparate range of pathogens all conferring similar risk, the likely causal factor is exposure to the maternal immune response. This is supported by the wide range of maternal stressors associated with schizophrenia. Spousal bereavement \citep{fagundes_spousal_2018, jones_prospective_2015}, chronic stressors such as unemployment and caregiving \citep{kiecolt-glaser_childhood_2011, cohen_chronic_2012, kiecolt-glaser_chronic_2003}, and traumatic stress \citep{teche_resilience_2017, passos_inflammatory_2015} are all associated with higher maternal proinflammatory biomarkers and immune dysfunction. Exposure to similar traumatic life events has also been shown to increase risk of neuropsychiatric disorder in offspring \citep{khashan_higher_2008, susser_latent_2008, malaspina_acute_2008}. It is likely that the mediating factor for the effect of prenatal infection and other maternal stressors is a maternal proinflammatory immune response.

This has been well documented in animal models, in which immunogens such as the viral mimetic poly(I:C) given to pregnant dams leads to adverse neurodevelopment in the offspring \citep{brown_maternal_2018}. These have been crucial in uncovering some of the biological processes underpinning the effect of maternal immune activation. Changes to brain structure and morphology as well as neuronal density have been identified \citep{silveira_effects_2017, aavani_maternal_2015, canetta_maternal_2016, shin_yim_reversing_2017}, which may be due to impaired neuronal migration during neurodevelopment \citep{oskvig_maternal_2012, harvey_stereological_2012, meyer_adult_2008}. These findings are consistent with the human neuropathology of schizophrenia, where MRI brain imaging and immunohistochemical analysis have identified similar structural alterations and neuronal densities \citep{arnone_magnetic_2009, olabi_are_2011, smiley_hemispheric_2011}. 

One underlying cause of this may be microglial priming induced by maternal immune activation. Microglia play a key role in neurodevelopment, facilitating neuronal migration, synaptic pruning and synaptogenesis \citep{}. Maternal immune activation has been shown to impact microglial activity in several different ways. Tangential microglial migration is delayed in animal models, while altered microglial activation has been observed as late as postnatal day 180, suggesting a long-lasting effect of prenatal immune activation on microglial morphology \citep{van_den_eynde_hypolocomotive_2014, zhang_maternal_2018}. These persistent activated microglia may shape the homeostasis of the brain with significant impact on neurodevelopment. This is known as the neuroinflammation hypothesis of schizophrenia, where the cytotoxic effects of persistent pro-inflammatory microglia and associated growth factors impairs neurogenesis and synaptic function and leads to psychiatric disorder \citep{laskaris_microglial_2016, monji_cytokines_2009}. However the question remains, what molecular and cellular processes are leading to this lasting neuroinflammation? The answer may lie in epigenetic changes introduced during pregnancy by prenatal insult. Epigenetic mechanisms serve to regulate gene expression without altering the genome sequence. DNA methylation is the addition of a methyl group to a gene sequence which, when located in a promoter region, acts to suppress gene transcription and thus lower expression of that gene. Methylation changes have also been shown to mediate the proinflammatory activation of microglial cells \citep{carrillo-jimenez_tet2_2019}. It is possible that this is the mediating factor between maternal immune activation and the associated permanent alteration to the trajectory of neurodevelopment. 

This paper will attempt to explore this link, by combining data from two similar studies of maternal immune activation which identified DNA methylation changes and gene expression differences in a mouse model. These two datasets will be analysed for overlap at the levels of individual genes, enriched biological pathways, and enriched cell types. Since an increase in DNA methylation leads to a decrease in gene expression and vice versa, i.e. an inverse relationship, it will be crucial to separate the two datasets by direction of effect, i.e. upregulated and downregulated gene expression, and hypermethylated and hypomethylated. The overlap between the corresponding direction will then be analysed. By analysing any overlap between these two datasets, it may be possible to establish a causal link between DNA methylation and adverse neurodevelopment.

\end{introduction}



\begin{methods}
\section{Methods}

\subsection{Preparation of Raw Data}
This analysis was based on data published in two papers; "Genome-wide DNA Methylation Changes in a Mouse Model of Infection-Mediated Neurodevelopmental Disorders" \citep{richetto_genome-wide_2017-1} in which the authors performed genome-wide bisulfite sequencing, and "Genome-Wide Transcriptional Profiling and Structural Magnetic Resonance Imaging in the Maternal Immune Activation Model of Neurodevelopmental Disorders" \citep{richetto_genome-wide_2017}, which conducted microarray expression analysis. These two papers boast very similar experimental design. Each used a murine model of maternal immune activation with C57Bl6/N mice. Each model featured as an immunogen an intravenous injection of 5mg/kg viral mimetic poly(I:C), with 0.9 saline as control. Both studies administered the poly(I:C) at gestation day (GD) 17. Both studies contained sequence samples taken from  the prefrontal cortex of the adult offspring, aged roughly over 12 weeks. Expression data was obtained from the NCBI GEO depository (GEO accession number GSE77973). As methylation raw data was unavailable, the list of differentially methylated genes given in supplementary materials to the methylation paper was used.

\subsection{Differential Expression Analysis}
Differential expression analysis was conducted using the R package Limma \citep{ritchie_limma_2015}. Limma is designed for processing gene expression data, including RNA-seq and microarray data, using linear modelling. The 6 vehicle and 6 control samples were normalised, then analysed for differentially expressed genes. Significance thresholds of p < 0.05 and the absolute fold change (log2) 0.137 were set, yielding 211 downregulated and 446 upregulated genes.

\begin{figure}[htbp]
\centerline{\includegraphics[width = \linewidth]{Expression_heatmap}}
\caption{Heatmap detailing relative gene expression levels for each sample. Control samples are the first six columns on the left, while vehicle samples are the six columns on the right. Rows are the genes represented in the dataset. Blue indicates downregulated expression compared to control, while yellow indicates upregulated expression.}\label{fig:02}
\end{figure}

%\subsection{Differential Methylation Analysis}
%The raw methylation data was not made available by the publishers. In lieu of the raw data, the list of differentially methylated genes identified by the authors at GD17 at single-base resolution was used.  

\subsection{Pathway Enrichment}
Pathway enrichment analysis was conducted using the online software tool GProfiler \citep{raudvere_gprofiler_2019} (biit.cs.ut.ee/gprofiler/gost), with results confirmed using secondary software tool PantherDB \citep{thomas_panther_2003} (pantherdb.org). Results were restricted to GO terms pertaining to biological process. All results underwent FDR correction for multiple testing with a statistical threshold of 0.05.

Pathway enrichment was conducted on the lists of differentially methylated and differentially expressed genes. These gene lists were split by direction of effect; downregulated and upregulated genes, and hypermethylated and hypomethylated genes. The resulting pathways were tested for any overlap between results from downregulated and hypermethylated genes, and then upregulated and hypomethylated genes, to confirm an inverse relationship and therefore a concordant effect of DNA methylation and gene expression on affected biological pathways.

\subsection{Cell Type Enrichment}
This step of the analysis was conducted using the EWCE package for R \citep{skene_identification_2016}. This software package is designed to perform expression-weighted enrichment using a single-cell expression dataset as reference. A gene set that is more highly expressed in a cell type than by chance is considered enriched for that cell type, implying greater importance of the gene set to that cell type's activity and function. Single-cell expression data and cell type annotations were taken from mousebrain.org, an online data resource accompanying the publication by Zeisel et al \citep{zeisel_molecular_2018}. This dataset contains 265 cell types from a comprehensive range of brain tissue samples including from the crtexx, hippocampus, striatum, as well as peripheral and enteric nervous systems. A detailed diagram showing the tissues and cell types represented in this dataset, as well as a dendrogram of cell type relationship, is given in Figure~1\vphantom{\ref{fig:01}}.  The lists of differentially methylated and expressed genes, separated by direction of effect, were tested for enrichment of cell types. The authors of the EWCE package recommend above 10,000 bootstrapping random gene lists for publication quality; 100,000 were used. This generated expression-weighted enrichment and a significance value for each cell type, indicating which cell types each gene list is more highly expressed in. The plotting function provided in the EWCE package was lacking for the purposes of this analysis, so was modified (this is provided at the source Github repository). Cell types were plotted against standard deviations away from mean enrichment, with FDR corrected significance values.

\end{methods}

\begin{results}
\section{Results}

\subsection{Differentially Expressed Genes}
Differential expression analysis revealed 657 differentially expressed genes, 446 of which were upregulated and 211 downregulated. An expression heatmap detailing these results is shown in Figure~2\vphantom{\ref{fig:02}}. A comprehensive list of differentially expressed genes can be found in supplementary table 1. 

\begin{figure}[htbp]
\centerline{\includegraphics[width = \linewidth]{Gene_overlap}}
\caption{Visualisation of the genes overlapping between the methylation and expression datasets. MGI gene IDs are given on the X axis, and fold change is given on the Y axis. For each gene, its methylation and expression fold change are plotted to illustrate their direction of effect and determine if there is a concordant effect of maternal immune activation on each condition. Genes for which there is a concordant effect and a possible link between methylation and expression are given in blue, while genes without a likely link are given in red.}\label{fig:03}
\end{figure}

\subsection{Differentially Methylated Genes}
There were 1,489 overall differentially methylated genes. 933 of these were hypomethylated, while 556 were hypermethylated. A comprehensive list of these genes can be found in supplementary table 2.

\begin{figure}[!htb]
\centerline{\includegraphics[width = 0.9\linewidth]{downreg_pathways}}
\caption{Gene ontology relationship tree of overlapping pathways enriched for downregulated and hypermethylated genes. Terms in yellow are the identified enriched pathways, while terms in white are related processes added during construction of the tree. Produced using QuickGO published by the EBI \citep{binns_quickgo_2009} (www.ebi.ac.uk/QuickGO).}\label{fig:04}
\end{figure}

\subsection{Overlapping Genes Relate to Neurodevelopmental and Immune Function}
There were 27 overlapping genes between the differentially expressed and differentially methylated lists of genes. A list of these genes as well as significance and fold change values is given in supplementary table 3. To further examine any potential causal relationship, overlapping genes were checked for the direction of effect for the two conditions (shown in Figure~3\vphantom{\ref{fig:03}}). An inverse relationship, i.e. a concordant effect, would mean a downregulated gene is also hypermethylated, and vice versa. This would indicate a possible causal effect of methylation changes on changes in gene expression.  For 19 of the 27 genes, there was an inverse relationship between DNA methylation and gene expression, i.e. a concordant effect, while for 8 there was not, i.e. a discordant effect. For example, in figure 1 the gene Cldn11 has a negative fold change for gene expression and is therefore downregulated, while its methylation fold change is positive so is hypermethylated, meaning there is an inverse or concordant relationship between the two for this gene. A concordant relationship indicates a potential causal link for these 19 genes, that altered DNA methylation patterns could be driving altered gene expression into adulthood.

Many of these genes are crucial for neurodevelopment and immune functionin mice. Dysfunction of the Galt gene leads to abnormal Purkinje cell morphology and brain inflammation \citep{tang_subfertility_2014}. Hcfc2 is a transcription factor with an associated disease phenotype of decreased TNF- and interferon-B secretion \citep{sun_hcfc2_2017}. Similarly Map4k3 dysfunction, a protein kinase involved in TNF-\textalpha signalling, is associated with decreased secretion of IL-2, IL-4, and IL-17, as well as abnormal T\textsubscript{H}1 and T\textsubscript{H}2 cell differentiation \citep{chuang_kinase_2011}. Dysfunction of the Slc9a9 gene is interestingly associated with abnormal CNS synaptic transmission, abnormal neuron physiology, as well as several abnormal social behaviour phenotypes \citep{yang_autism_2016, ullman_mouse_2018}.

There is less evidence for the importance of the human orthologs in neurodevelopment, although many are still involved in important processes. CLDN11 is a gene expressed predominantly in oligodendrocyte cells, with a crucial role in myelination as well as regulation of oligodendrocyte proliferation and migration \citep{tiwari-woodruff_ospclaudin-11_2001}. Hypermethylation of this gene is here linked to its downregulated expression, suggesting a role in adverse neurodevelopment. LMO4 plays a role in spinal cord neuron differentiation and cell migration. Its hypomethylation is linked here to upregulated expression, suggesting its hypomethylation may be linked to neuroanatomical changes. It is noteworthy that a majority of the overlapping genes have a similar direction of effect between differential methylation and expression, and that many play a role in neurodevelopment or have been implicated in psychiatric disorder. However it should be noted that the number of overlapping genes is still a little low; 27 genes out of 657 differentially expressed and 1489 differentially methylated genes.

\begin{figure}[!htb]
\centerline{\includegraphics[width = 0.5\linewidth]{Upreg_pathways}}
\caption{Gene ontology relationship tree of overlapping pathways enriched for upregulated and hypomethylated genes. Produced using QuickGO published by the EBI \citep{binns_quickgo_2009} (www.ebi.ac.uk/QuickGO).}\label{fig:05}
\end{figure}

\subsection{Neurodevelopmental Pathways are Affected by Both Methylation and Expression}
To further investigate the link between the two, the gene lists were divided based on their direction of effect, and tested for pathway enrichment. The results were then tested for any overlap based on direction of effect (shown in figure~4\vphantom{\ref{fig:04}} andfigure~5\vphantom{\ref{fig:05}}). Full lists of the overlapping enriched terms are given in supplementary tables 4 and 5. 

Pathways that were both downregulated and hypermethylated seemed to have a strong relationship to anatomical development and. These also included many pathways relating to neurodevelopment; generation of neurons, regulation of neurogenesis and neuron differentiation, and nervous system development. In the case of upregulated and hypomethylated pathways however, there were only 4 overlapping, all of which related to cellular component assembly and none to neurodevelopment. This analysis makes clear that there is some connection between the biological processes affected by DNA hypermethylation and downregulated gene expression, while there is likely no such connection between hypomethylated and upregulated biological processes. Prenatal hypermethylation may be the process driving downregulation of biological pathways related to neurogenesis, and thus affecting the outcome of neurodevelopment. However the same is not likely to be true for hypomethylation.


\begin{figure*}[!h]
\centerline{\includegraphics[width = \linewidth]{EWCE}}
\caption{Expression-weighted cell enrichment analysis. Graphs labelled A, B, C, and D, are the enrichment patterns for each gene set, which is specified in the graph titles. Each bar represents an enrichment score for a given cell type. Enrichment scores that also reach the FDR significance threshold are marked with an asterisk. Bars are coloured and ordered according to their cell type taxonomy. As a guide, the dendrogram from  figure~1\vphantom{\ref{fig:01}} that outlines the cell types and their taxonomy is given in the centre of each pair of graphs.}\label{fig:06}
\end{figure*}


\subsection{Oligodendrocyte Function and Neurogenesis are Likely Impaired by Maternal Immune Activation}
Results from the EWCE analysis (shown in figure~6\vphantom{\ref{fig:06}}) showed statistically significant enrichment of downregulated genes for oligodendrocyte and astroependymal cells, specifically ependyma in the dorsal midbrain, and subventricular radial glia-like cells. There is a similar pattern of enrichment for hypermethylated genes. These same cell types were the most highly and significantly enriched, suggesting a possible link between DNA methylation and altered gene expression in these cell types. However, it must be noted that cell type enrichments for hypermethylated genes survived FDR correction for multiple testing. The similar patterns of enrichment is a promising result, however it is not enough to draw any definitive conclusions. Comprehensive enrichment results are given in supplementary tables 6-9.

Upregulated genes showed a pattern of enrichment largely skewed towards CNS neurons, particularly neurons in the cortex and hippocampus. However, the only enriched cell type that survived FDR correction was a excitatory glutamatergic neuronal cell type located in the midbrain. Hypomethylated genes followed a similar pattern of enrichment, skewed towards CNS neurons, however there were no cell type associations that survived FDR correction for multiple testing. Once again there is a link hinted at between the enrichment patterns of both gene sets.

\end{results}

\begin{discussion}
\section{Discussion}
The preliminary differential expression analysis identified 657 overall differentially expressed genes. This is in contrast to the findings identified in the original study \citep{richetto_genome-wide_2017}. They identified a total of 116 differentially expressed genes with 55 downregulated and 61 upregulated. A random sample of genes in common found similar fold change and p values in each dataset. However, when checking the total genes in common, it was found that 83 of the 116 differentially expressed genes were identified in this analysis as well as by the original authors, while 33 were not. Based on these two pieces of evidence, it's likely that the disparity is caused by differences in analytical methodology between Limma and the Partek suite used by the original authors, which is proprietary.

The results from each of the three stages of analysis show some promise for the methylation hypothesis. Overlapping genes between the two datasets showed largely similar direction of effect, i.e. that methylation of a given gene induced by maternal immune activation leads to decreased expression of that gene. This was the case for the majority of the overlapping genes. Present in the overlapping genes were many critical to neurodevelopment and normal neuronal activity, as well as social behaviour, and normal immune function. The presence of genes relating to immune function such as Hcfc2, Map4k3, and Galt, suggest some persistent immunological impact which is an interesting result considering the role of neuroinflammation in schizophrenia \citep{laskaris_microglial_2016}. CLDN11, which was both hypermethylated and downregulated, plays a key role in myelination and oligodendrocyte activity; its dysfunction has been linked to schizophrenia \citep{cassoli_disturbed_2015, tkachev_oligodendrocyte_2003, dracheva_myelin-associated_2006} and leads to a schizotypal phenotype in mice \citep{maheras_absence_2018}. LMO4 plays a key role in neurodevelopment through neural tube closure and spinal cord neuron differentiation. It also has an inverse relationship between methylation and expression in this study, suggesting it may play a role in altered neurodevelopment through differential methylation. There may be a stronger link between the genes identified and adverse neurodevelopment, and methylation may play a key role, however further study and more concrete functional analysis would need to confirm this.

There was a sizable number of overlapping pathways enriched for downregulated and hypermethylated genes, many of which involved in tissue developmental processes and some involved directly with neurogenesis and neurodevelopment. This suggests that neurodevelopmental biological processes may be downregulated as a result of DNA methylation changes. There was less overlap between upregulated and hypomethylated pathways, none of which were involved in any neurological process. The results suggest that hypermethylation has some effect on neurodevelopment by altering gene expression, while hypomethylation has no such impact. These results are encouraging, and further functional analysis into the role of methylation on neurodevelopmental pathways would prove promising.

Given the fact that oligodendrocyte and astroependymal cells are enriched for both hypermethylated and downregulated genes, it is plausible that their activity is inhibited as a result of persistent methylation changes. It is likely then that myelination and neurogenesis are impaired in adulthood as a result of maternal immune activation. 

Brain dysconnectivity caused by disruptions to myelination by oligodendrocytes is a leading hypothesis in the study of schizophrenia \citep{cassoli_disturbed_2015}. Oligodendrocyte dysfunction is well known to cause aberrant myelination with far-reaching consequences to cognitive and neural functions \citep{fields_white_2008}. Functional evidence including brain imaging and immunohistochemical studies has established reduced oligodendrocyte density in the prefrontal cortex, hippocampus, and other brain regions in schizophrenia patients \citep{uranova_oligodendroglial_2004, schmitt_stereologic_2009, vostrikov_deficit_2007}. Radial glial cells in the subventricular zone are the primary progenitor cells during embryonic neurodevelopment for neurons in the cerebral cortex, as well as some glia including oligodendrocytes and astrocytes. These cells also perform neurogenesis in the adult brain [martinez-cerdeno 2018], which in a previous model of maternal immune activation was shown to be impaired with a decrease in neural stem cells and new neurons \citep{liu_effects_2013}. There is also strong functional evidence that dysfunctional adult neurogenesis contributes to the pathology of schizophrenia \citep{iannitelli_schizophrenia_2017}. 

A possible connection is microglial activation. Activated microglia and altered microglial density has been repeatedly identified in animal models and post-mortem study of schizophrenia patients. Microglial activation has also been shown to damage oligodendrocytes during neurodevelopment. As posited by \cite{cassoli_disturbed_2015}, they may alter oligodendrocyte activity by secreting pro-inflammatory cytokines and growth factors leading to impaired connectivity in the developing brain. This altered homeostasis may also underlie dysfunctional neurogenesis which is a hallmark of neurodevelopmental disorder and is seen in this animal model. Epigenetic changes including histone modifications, microRNA expression, and DNA methylation, are important mediators of microglial plasticity \citep{}, although much more research has been conducted into the role of histone modification and microRNA activity. Age-dependent increase in levels of microglial IL-1\beta has been associated with hypomethylation of its promoter region \citep{matt_aging_2016}, however there is a lack of evidence that global DNA methylation patterns play a role in altering microglial plasticity in disease \citep{coppieters_global_2014, phipps_neurofilament-labeled_2016}. It should also be noted that at no analysis level in this study were microglial cells or related processes identified; no gene list is enriched for microglial cell types. Persistent microglial activation may cause the effects of maternal immune activation, but if DNA methylation changes play a mediating role it is clear that it does not directly impact microglial activity, and methylation changes in some other biological process may be triggering the pro-inflammatory microglial profile.

There is a good deal of evidence in this study for the role of DNA methylation mediating changes in gene expression caused by maternal immune activation. Overlapping genes were identified between differentially methylated and expressed genes, many of which with a role in neurodevelopment. Pathways involved in neurogenesis and neurodevelopment were found to be enriched for both downregulated and hypermethylated genes, suggesting downregulated neurodevelopment driven by altered methylation. Oligodendrocyte and astroependymal cells were found to be enriched for both downregulated and hypermethylated genes, suggesting that hypermethylation of genes crucial to these cell types are suppressing their expression with an impact on their neurodevelopmental activity. However, more analysis both functional and computational will be helpful to further elucidate the role of DNA methylation mediating the impact of maternal immune activation.

\end{discussion}

\begin{conclusion}
\section{Conclusion}
Epigenetic changes introduced by maternal immune activation almost certainly play some role in mediating its effects. This study supports this conclusion, as does a wealth of previous study evidence \citep{richetto_genome-wide_2017-1, tang_epigenetic_2013, labouesse_maternal_2015, basil_prenatal_2014}. The question remains, whether methylation changes explain the observed phenotypes and the extent to which they impact neurodevelopment. A commentary on the DNA methylation study used in this analysis published in Biological Psychiatry, concluded that the author's analysis supported a link between methylation changes and neurodevelopmental disorder \citep{kundakovic_fearing_2017}.

However, many questions remain including the role of gestational timing of MIA, assessment of other brain regions, other forms of epigenetic modification, and more. Based on the evidence presented in this analysis, there is likely a link between DNA methylation changes and altered gene expression brought about by maternal immune activation, with a putative causal role in associated abnormal neurodevelopment. However many uncertainties remain. Data from a single animal model would be much more effective than combined datasets in identifying any synergy between methylation and expression. Other design features including additional brain regions and gestational timings of MIA should be explored, as well as the role of other epigenetic features. Functional analysis into the role of altered methylation of specific genes or pathways could potentially prove a causal role. DNA methylation undoubtedly has some part to play in mediating the impact of maternal immune activation. With further research, and a broader range of analysis, its role may become clear.
\end{conclusion}

\bibliographystyle{natbib}
\bibliography{document}

\end{document}